common/perception_utils/include/perception_utils/object_classification.hpp

```diff
@@ -92,6 +93,82 @@ inline bool isLargeVehicle(const std::vector<ObjectClassification> & object_clas
   auto highest_prob_label = getHighestProbLabel(object_classifications);
   return isLargeVehicle(highest_prob_label);
 }
+
+inline uint8_t toLabel(const std::string & class_name)
+{
+  if (class_name == "UNKNOWN") {
+    return ObjectClassification::UNKNOWN;
+  } else if (class_name == "CAR") {
+    return ObjectClassification::CAR;
+  } else if (class_name == "TRUCK") {
+    return ObjectClassification::TRUCK;
+  } else if (class_name == "BUS") {
+    return ObjectClassification::BUS;
+  } else if (class_name == "TRAILER") {
+    return ObjectClassification::TRAILER;
+  } else if (class_name == "MOTORCYCLE") {
+    return ObjectClassification::MOTORCYCLE;
+  } else if (class_name == "BICYCLE") {
+    return ObjectClassification::BICYCLE;
+  } else if (class_name == "PEDESTRIAN") {
+    return ObjectClassification::PEDESTRIAN;
+  } else {
+    return ObjectClassification::UNKNOWN;
+  }
```

Comment:
This handling looks not good.
If we have typo on class types as a string, the type will be unknown.

I think it's okay to throw exception in this case.

Reply:
That's certainly true. Thank you for recommendation.
Fixed at [15d8d0a](https://github.com/autowarefoundation/autoware.universe/pull/2090/commits/15d8d0afa3242d3a7120326757d49cc428a73bf3).